packages/components/addon/components/hds/app-footer/legal-links.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
+export default class HdsAppFooterIndexComponent extends Component {
+  get termsUrl() {
+    return (
+      this.args.termsUrl ??
+      'https://portal.cloud.hashicorp.com/terms-of-service'
+    );
+  }
```

Comment:
Q: How configurable should the LegalLinks and what would be the best API? Should I include display options to turn off individual legal links? Should I expose text and urls for all (or more of) the links?

Reply:
I think we should base this on the audit we already have. we can always extend them later, exposing new arguments (it would not be a breaking change)

Reply:
Ok, I have exposed all the link href values for now as you suggested above.